frontend/javascripts/oxalis/view/left-border-tabs/layer_settings_tab.js

```diff
@@ -740,13 +741,24 @@ class DatasetSettings extends React.PureComponent<DatasetSettingsProps, State> {
               value={userConfiguration.particleSize}
               onChange={this.onChangeUser.particleSize}
             />
-            <NumberSliderSetting
-              label={settings.clippingDistanceArbitrary}
-              min={userSettings.clippingDistanceArbitrary.minimum}
-              max={userSettings.clippingDistanceArbitrary.maximum}
-              value={userConfiguration.clippingDistanceArbitrary}
-              onChange={this.onChangeUser.clippingDistanceArbitrary}
-            />
+            {this.props.isArbitraryMode ? (
+              <NumberSliderSetting
+                label={settings.clippingDistanceArbitrary}
+                min={userSettings.clippingDistanceArbitrary.minimum}
+                max={userSettings.clippingDistanceArbitrary.maximum}
+                value={userConfiguration.clippingDistanceArbitrary}
+                onChange={this.onChangeUser.clippingDistanceArbitrary}
+              />
+            ) : (
+              <LogSliderSetting
```

Comment:
Yes, I restored the old code for that slider from before the panel reorganization. See here: https://github.com/scalableminds/webknossos/pull/5384/files#diff-a803b9265f591aeee5b68bbe94a8848932e756a68869dccfa79c58f6b4ae77c0L164
It would probably make sense to use the LogSlider in both cases, but sticking to the old behavior seemed to be the most straightforward solution to me for now.